apps/web/components/apps/layouts/AppsLayout.tsx

```diff
@@ -22,7 +22,8 @@ export default function AppsLayout({ children, actions, emptyStore, ...rest }: A
 
   if (session.status === "loading") return <></>;
   return (
-    <ShellMain {...rest} actions={actions?.("block")} hideHeadingOnMobile>
+    <ShellMain {...rest} actions={actions?.("block")} withoutSeo hideHeadingOnMobile>
+      <HeadSeo title="App Store" description="Connecting people, technology and the workplace." />
```

Comment:
I think we should use i18n here.

Reply:
and other pages as well

Reply:
I was not using i18n, as @PeerRich suggested. Let me quickly fix that.

Reply:
Okay

Reply:
I don't understand why we are passing withoutSeo to ShellMain? Doesn't ShellMain support for SEO work correctly?

Reply:
@hariombalhara ShellMain doesn't support SEO, and if we don't provide withoutSeo, then ShellMain uses the heading as the title and the subtitle as the description. So basically, withoutSeo is preventing that.

Reply:
@hariombalhara yeah that's useless.